### DISCUSSION TITLE
Am getting this bug in foocus someone help please, how to get rid of it any help will be appriciated.

### DISCUSSION THREAD

**Discussion by gauravsparty** [lllyasviel/Fooocus#2974] [Q&A] [answered]
![error](https://github.com/lllyasviel/Fooocus/assets/149142063/f25b36cb-9f77-428d-9c20-a6f923ed73d2)

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/issues/2062 / https://github.com/lllyasviel/Fooocus/issues/2840

tl;dr: make sure you use 7zip to extract the zip file + check the comments in the issue https://github.com/lllyasviel/Fooocus/issues/2062 if this didn't help
